### PR TITLE
[stable/kapacitor] Chart deprecation

### PR DESCRIPTION
**Pull request by naseemkullah**
As per https://github.com/helm/charts/blob/master/PROCESSES.md cc @davidkarlsen @Aisuko 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/kapacitor/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 name: kapacitor
-version: 1.2.1
+version: 1.2.2
 appVersion: 1.5.2
-description: InfluxDB's native data processing engine. It can process both stream
+deprecated: true
+description: DEPRECATED InfluxDB's native data processing engine. It can process both stream
```
Comment (davidkarlsen) [inline stable/kapacitor/Chart.yaml:6]: any new home to point to?

Comment (naseemkullah) [reply stable/kapacitor/Chart.yaml:6]: Yep: https://github.com/helm/charts/tree/master/stable/kapacitor#deprecated---kapacitor

Already in the README from previous commit.

**Review by Aisuko — APPROVED** [review]
/approve

**Pull request merged**